base/file.jl

```diff
@@ -381,13 +393,14 @@ function mktemp(fn::Function, parent=tempdir())
 end
 
 """
-    mktempdir(f::Function, parent=tempdir())
+    mktempdir(f::Function, parent=tempdir(); prefix="$(repr(temp_prefix))")
```

Comment:
I think this will include two sets of quotes (quotes are included in the output of `repr` of a string).

Reply:
See previous comment: https://github.com/JuliaLang/julia/pull/23634#discussion_r142481786